index.js

```diff
@@ -1,32 +1,62 @@
-function ratioToRGB ({ r, g, b }) {
+function ratioToRGB({ r, g, b }) {
   return ({
     r: r * 255,
     g: g * 255,
-    b: b * 255
+    b: b * 255,
   })
 }
 
-export default function getColors (document, nodeType, nameRegex) {
+function ratioToRGBA({ r, g, b, a }) {
+  return ({
+    r: r * 255,
+    g: g * 255,
+    b: b * 255,
+    a: a
+  })
+}
+
+export default function getColors(document, channelType, nodeType, nameRegex) {
   nodeType = nodeType || ['RECTANGLE']
   nameRegex = nameRegex || /.*/g
+  channelType = channelType || 'rgb'
 
   if (!document) {
     return null
   }
 
   let results = []
 
-  function visit (node) {
+  function visit(node) {
+
     if (
       node.name.match(nameRegex)
       && nodeType.includes(node.type)
       && node.fills.length
       && node.fills[0].color
     ) {
-      const color = ratioToRGB(node.fills[0].color)
+
+      let color;
+
+      if (channelType === 'rgba') {
+
+        let newColorObj = {
```

Comment:
This could be written as
```
const newColorObj = {
  ...node.fills[0].color,
  a: node.fills[0].opacity
}
``` and should probably use `const` as well since we're not changing this value :)